### PR TITLE
Add Puppeteer to headless post

### PR DESCRIPTION
**Pull request by ebidel**

### CODE REVIEW

File: src/content/en/updates/2017/04/headless-chrome.md [hunk 1]
```diff
@@ -140,9 +140,66 @@ commands going across the wire, communicating with the browser.
...
-### Launching Chrome {: #nodelaunch }
+### The Puppeteer API {: #puppeteer }
 
-In the previous section, we [started Chrome manually](#cli) using
+[Puppeteer](https://github.com/GoogleChrome/puppeteer) is a Node library
+developed by the Chrome team. It provides a high-level API to control headless
+(or full) Chrome. It's similar to other automated testing libraries like Phantom
+and NightmareJS, but it only works with the latest versions of Chrome.
+
+Among other things, Puppeteer can be used to easily screenshots, create PDFs,
```
Comment (jpmedley) [inline src/content/en/updates/2017/04/headless-chrome.md]: 'to easily screenshots' to 'to easily take screenshots'.

File: src/content/en/updates/2017/04/headless-chrome.md [hunk 2]
```diff
@@ -140,9 +140,66 @@ commands going across the wire, communicating with the browser.
...
+navigate pages, and fetch information about those pages. I recommend the library
+if you want to get up and running quickly with automated browser testing.
```
Comment (jpmedley) [inline src/content/en/updates/2017/04/headless-chrome.md]: Change

'if you want to get up and running quickly with automated browser testing'

to

'if you want to quickly automate browser testing'

File: src/content/en/updates/2017/04/headless-chrome.md [hunk 3]
```diff
@@ -140,9 +140,66 @@ commands going across the wire, communicating with the browser.
...
+It hides away the complexities of the DevTools Protocol and takes care of
```
Comment (jpmedley) [inline src/content/en/updates/2017/04/headless-chrome.md]: Protocol should be lower case.

File: src/content/en/updates/2017/04/headless-chrome.md [hunk 4]
```diff
@@ -140,9 +140,66 @@ commands going across the wire, communicating with the browser.
...
+redundant tasks like launching a debug instance of Chrome.
+
+Install it:
+
+    yarn add puppeteer
```
Comment (jpmedley) [inline src/content/en/updates/2017/04/headless-chrome.md]: Is yarn common enough to make this the default install? Wouldn't npm be better?

File: src/content/en/updates/2017/04/headless-chrome.md [hunk 5]
```diff
@@ -140,9 +140,66 @@ commands going across the wire, communicating with the browser.
...
+```
+
+Check out [Puppeteer's documentation](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md)
+to learn more about the full API.
+
+### The CRI library {: #cri }
+
+[chrome-remote-interface](https://www.npmjs.com/package/chrome-remote-interface)
+is a lower-level library than Puppeteer's API. I recommend it if you want to be
+close to the metal and use the [DevTools Protocol][dtviewer] directly.
```
Comment (jpmedley) [inline src/content/en/updates/2017/04/headless-chrome.md]: Protocol should be lower case.

File: src/content/en/updates/2017/04/headless-chrome.md (hunk 1 above)
Comment (ebidel) [inline src/content/en/updates/2017/04/headless-chrome.md]: done

File: src/content/en/updates/2017/04/headless-chrome.md (hunk 3 above)
Comment (ebidel) [inline src/content/en/updates/2017/04/headless-chrome.md]: done throughout

File: src/content/en/updates/2017/04/headless-chrome.md (hunk 4 above)
Comment (ebidel) [inline src/content/en/updates/2017/04/headless-chrome.md]: yea. I'm already using it everywhere in the post.

File: src/content/en/updates/2017/04/headless-chrome.md (hunk 5 above)
Comment (ebidel) [inline src/content/en/updates/2017/04/headless-chrome.md]: done

**Comment by ebidel** [comment]
PTAL

**Pull request merged**